docs/analytics/cloud.mdx

```diff
@@ -11,8 +11,101 @@ To enable Sourcegraph Analytics:
 - Create an account on [Sourcegraph Accounts](https://accounts.sourcegraph.com/), or find the email address associated with your existing account.
 - Contact your Sourcegraph Technical Advisor or point of contact (or email us at support@sourcegraph.com if you don't know your contact), provide them with the email address you used to register above and ask for access to Sourcegraph Analytics.
 - They will validate your account and link it to your Sourcegraph Enterprise instance's usage metrics.
-- Sign in to [Sourcegraph Analytics](https://cody-analytics.sourcegraph.com).
+- Sign in to [Sourcegraph Analytics](https://analytics.sourcegraph.com).
 
 ## Data export
 
 Sourcegraph Cloud Analytics also includes a CSV export option with key metrics like the number of searches, chats, autocomplete suggestions, completion acceptance rate (CAR %), and more. The data is split by user, day, client/editor, and programming language, perfect for automating retrieval and analyzing data in ways that make the most sense to your organization.
+
+
+## Access Tokens
+
+For Sourcegraph Analytics, you can generate an access token for programmatic access. Tokens are long lived with an optional expiry and have the same permissions to access instance data as the user who created them.
+
+### Token management APIs
+
+Token management is currently only available via the Sourcegraph Analytics UI.
```

Comment:
```suggestion
Token management is currently only available via the Sourcegraph Analytics API.
```

We don't have a UI yet right? We could also include a note here that UI-based management is on the roadmap (abstractly)